### ISSUE TITLE
Access and roles with group security force users to logout

### ISSUE THREAD

**Issue by darthtang** [salesagility/SuiteCRM#702]
Scenario 

The company has multiple users logged in at once 

The system administrator users into groups and set the permissions of the groups with role access.

The changes will not apply 

Enhancement suggestion->when this scenario happens then get all users to force logout then log back in to enable the changed security settings

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Access and roles with group security force users to logout](https://trello.com/c/gSAXBKaF/134-access-and-roles-with-group-security-force-users-to-logout)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
